frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature

```diff
@@ -1,54 +1,36 @@
 
-
-        Scenario: Deploy Database Application using Catalog Templates
-             When user selects Databases type
-              And user searches and selects Template card "MariaDB" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
-             Then user will be redirected to Topology page
-              And user is able to see workload "mariadb" in topology page
-
-
-        Scenario: Deploy Application using Languages Templates
-             When user selects Languages type
-              And user searches and selects Template card "Node.js + PostgreSQL (Ephemeral)" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
-             Then user will be redirected to Topology page
-              And user is able to see workload "nodejs-postgresql-persistent" in topology page
-
-
-        Scenario: Deploy MiddleWare Application using Catalog Templates
-             When user selects MiddleWare type
-              And user searches and selects Template card "Apache HTTP Server" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
-             Then user will be redirected to Topology page
-              And user is able to see workload "httpd-example" in topology page
-
-
-        Scenario: Deploy Application using Other types of Catalog Templates
-             When user selects Other type
-              And user searches and selects Template card "Nginx HTTP server and a reverse proxy" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
+    
+        @smoke
+        Scenario Outline: Deploy Application using different types of Catalog Templates
+             When user selects "<template_type>" from Templates type
+              And user searches and selects Template card "<card_name>" from catalog page
+              And user clicks Instantiate Template button on side bar
+              And user clicks create button
              Then user will be redirected to Topology page
-              And user is able to see workload "nginx-example" in topology page
+              And user is able to see workload "<workload_name>" in topology page
+
+        Examples:
+                  | template_type | card_name                             | workload_name             |
+                  | Databases     | MariaDB                               | mariadb                   |
+                  | Languages     | Node.js + PostgreSQL (Ephemeral)      | nodejs-postgresql-example |
+                  | Middleware    | Apache HTTP Server                    | httpd-example             |
+                  | Other         | Nginx HTTP server and a reverse proxy | nginx-example             |
```

Comment:
I think we can move this scenario to create-from-catalog.feature

Reply:
Updated